eZ/Publish/Core/Persistence/Cache/UrlWildcardHandler.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\Persistence\Cache;
+
+use eZ\Publish\API\Repository\Exceptions\NotFoundException as APINotFoundException;
+use eZ\Publish\Core\Base\Exceptions\NotFoundException;
+use eZ\Publish\SPI\Persistence\Content\UrlWildcard;
+use eZ\Publish\SPI\Persistence\Content\UrlWildcard\Handler as UrlWildcardHandlerInterface;
+
+class UrlWildcardHandler extends AbstractHandler implements UrlWildcardHandlerInterface
+{
+    /**
+     * Constant used for storing not found results for lookup().
+     */
+    private const NOT_FOUND = 0;
+
+    /**
+     * @see \eZ\Publish\SPI\Persistence\Content\UrlWildcard\Handler::create
+     */
+    public function create($sourceUrl, $destinationUrl, $forward = false)
+    {
+        $this->logger->logCall(
+            __METHOD__,
+            [
+                'sourceUrl' => $sourceUrl,
+                'destinationUrl' => $destinationUrl,
+                'forward' => $forward,
+            ]
+        );
+
+        $urlWildcard = $this->persistenceHandler->urlWildcardHandler()->create($sourceUrl, $destinationUrl, $forward);
+
+        $this->cache->invalidateTags(['urlWildcard-notFound']);
+
+        return $urlWildcard;
+    }
+
+    /**
+     * @see \eZ\Publish\SPI\Persistence\Content\UrlWildcard\Handler::remove
+     */
+    public function remove($id)
+    {
+        $this->logger->logCall(__METHOD__, ['id' => $id]);
+
+        $urlWildcard = $this->load($id);
+
+        $this->persistenceHandler->urlWildcardHandler()->remove($urlWildcard->id);
+
+        $this->cache->invalidateTags(['ez-urlWildcard-id-' . $urlWildcard->id]);
+    }
+
+    /**
+     * @see \eZ\Publish\SPI\Persistence\Content\UrlWildcard\Handler::load
+     */
+    public function load($id)
+    {
+        $this->logger->logCall(__METHOD__, ['id' => $id]);
```

Comment:
Wrong use here, `logCall ` is for non cached methods.

Should rather be something like:
```php
 if ($cacheItem->isHit()) {
     $this->logger->logCacheHit(['id' => $id]);

    return $cacheItem->get();
}

$this->logger->logCacheMiss(['id' => $id]);
```